### ISSUE TITLE
Staging Review finding: Inaccurate heading hierarchy on review page

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#91795]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Modernized Check-in
**Product:** eCheck-in
**Feature:** Mileage-only Travel Claim Submissions for VA Facilities that use Oracle Health EHR Software

## Findings details
**VA.gov Experience Standard - issue:** [User encounters content that does not meet best practices.](https://design.va.gov/about/experience-standards/#user-encounters-content-that-meets-best-practices)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @erinrwhite (IA), @briandeconinck (Accessibility)

### Description

The `Beneficiary travel agreement` heading on the review page is an h3, while other headings on the page are h2's. As an h3 it appears to be nested under the h2 `Starting address`, though conceptually they should be grouped separately.

<details><summary>Link, screenshot or steps to recreate</summary>

![heading-hierarchy-2](https://github.com/user-attachments/assets/82db3c4a-3648-40ac-9ab3-c2aab01140ee)

![heading-hierarchy-1](https://github.com/user-attachments/assets/3b7b30e1-cd8c-409a-b15a-bd2939d05e56)

</details>

## Recommended action

Adjust `Beneficiary travel agreement` to use an h2 heading.

## References

- **Accessibility Defect Severity:** N/A
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by brianseek** [comment]
@benbrasso-agile6 @zach-park @cgednalske this is merged and ready to be validated locally or by looking at the screenshots on the PR: https://github.com/department-of-veterans-affairs/vets-website/pull/31773

**Comment by benbrasso-agile6** [comment]
LGTM. @zach-park can you get this one updated in Figma too? This one can be updated in the prod Figma file as well. Check-in files.

**Comment by zach-park** [comment]
Annotation (e.g., H2 with H3 styling) is added on both Check-in and Production Figma files.

**Comment by zach-park** [comment]
> Annotation (e.g., H2 with H3 styling) is added on both Check-in and Production Figma files.

Annotation (e.g., H2 with H3 styling) is added on both Check-in and Production Figma files.

**Comment by zach-park** [comment]
> LGTM. @zach-park can you get this one updated in Figma too? This one can be updated in the prod Figma file as well. Check-in files.

Annotation (e.g., H2 with H3 styling) is added on both Check-in and Production Figma files.

**Comment by brianseek** [comment]
Thanks! closing
